package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "pegasus-feeds",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/main.js",
+  "scripts": {
+    "lint": "eslint ./src/**/*.ts ./test/**/*.ts --max-warnings=6",
+    "lint:fix": "eslint ./src/**/*.ts ./test/**/*.ts --fix",
+    "prettier": "prettier --config .prettierrc ./src/**/*.ts ./test/**/*.ts --check",
+    "prettier:format": "prettier --config .prettierrc ./src/**/*.ts ./test/**/*.ts --write",
+    "format": "npm run lint:fix && npm run prettier:format",
+    "start": "tsc src/main.ts && cd src && node main.js",
+    "restore": "tsc src/restoreFeeds.ts && cd src && node restoreFeeds.js",
+    "test": "tsc test/test.ts && cd test && node test.js"
```

Comment:
Please mention this in the readme. Consider calling this script an example instead of a proper test.